### PR TITLE
Add watch-youtube example

### PR DESCRIPTION
**Pull request by julianrubisch**

### CODE REVIEW

**Comment by catarak** [comment]
Our first Node For Max PR!!!! :tada: :tada: :tada:

This looks great and works great! I tested this on Mac and Windows. I'm gonna merge this in!

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ You might find an error in an example, or have a request for a example you would
 * [sockets](https://github.com/Cycling74/n4m-examples/tree/master/sockets): An example creating a web server with websockets. By [Cycling '74](https://github.com/Cycling74).
 * [tonal-chord-builder](https://github.com/Cycling74/n4m-examples/tree/master/tonal-chord-builder): Generate different chords based on a root note. By [Cycling '74](https://github.com/Cycling74).
 * [twitter](https://github.com/Cycling74/n4m-examples/tree/master/twitter): Communicate with Twitter from Max. By [Cycling '74](https://github.com/Cycling74).
+* [watch-youtube](https://github.com/Cycling74/n4m-examples/tree/master/watch-youtube): Communicate with Twitter from Max. By Julian from [Znibbles](https://www.znibbl.es).
```
Comment (caseypugh) [inline README.md:20]: FYI, this says `Twitter` here and it should say `YouTube`

Comment (julianrubisch) [reply README.md:20]: Damn... something always sneaks in... I‘ll revise!

Comment (catarak) [reply README.md:20]: ooo thank you! i can fix this quickly.

Comment (julianrubisch) [reply README.md:20]: That‘d be great! I think „watch YouTube in Max“ would be more descriptive

Comment (catarak) [reply README.md:20]: fixed! ✨

**Comment by catarak** [comment]
@julianrubisch i wanted to let you know that we've decided split this repository up into two repositories, one for Cycling '74 created and maintained examples, and one for community created and maintained examples. i linked to your example in the [community repository](https://github.com/Cycling74/n4m-community), to your forked n4m-examples repository. i would suggest making this example, and your static d3.js example into their own repositories, and updating the links in the community repository. let me know if you need help with any of this, and thanks for contributing examples!

**Pull request merged**